droplet_deployer/config.json

```diff
@@ -35,9 +35,19 @@
     }
   },
   "listeningPort": 5483,
-  "dropletSize": "1gb",
-  "imageId": 22557691,
-  "concentratedRegion": "lon1",
   "sshKeys": [],
-  "testMode": false
+  "testMode": false,
+  "provider": "digitalOcean",
+  "providerDetails": {
+    "digitalOcean": {
+      "snapshotId": "27270902",
+      "concentratedRegion": "lon1",
+      "size": "1gb"
```

Comment:
Size format according to D.O. API docs has now changed, as have the options available, so this line should now be:
"size": "s-1vcpu-2gb"

See https://developers.digitalocean.com/documentation/v2/#list-all-actions for a list of sizes in their new format.
Note that the s-1vcpu-2gb option is the lowest available when we use the droplet deployer snapshot (can demo manually if necessary)